functions/tab.fish

```diff
@@ -125,6 +128,8 @@ function __tab.term_program
         end
       else if [ "$VTE_VERSION" -ge 3803 -o "$COLORTERM" = "gnome-terminal" ]
         echo gnome_terminal
+      else if [ "$COLORTERM" = "xfce4-terminal" ]
+	echo xfce4_terminal
```

Comment:
&^%& tabstop not set right in vim. *Looked* OK, but there was a hidden /t. Code worked fine, but easy fix.